packages/@webex/internal-plugin-dss/test/unit/spec/dss.ts

```diff
@@ -1252,6 +1259,78 @@ describe('plugin-dss', () => {
         expect(Batcher.prototype.request.getCall(1).args).to.deep.equal(['id2']);
         expect(result).to.equal(response2);
       });
+
+      it('fails when mercury does not response but only the affected request, not all if them ', async () => {
```

Comment:
Had a look, I'll comment here instead. This test is just checking this particular function, so it may not need 4 requests or to wait 1000ms. It does need to check the timeout is passed through to the batcher correctly. You also need to add higher-level unit tests to '#lookup' to check that the timeout actually works properly (with testMakeBatchedRequests)